_data/social-media.yml

```diff
@@ -2,6 +2,10 @@
   link: https://www.facebook.com/PyConIndia/
   icon: assets/images/facebook-logo.svg
 
-- tile: twitter
+- title: twitter
   link: https://twitter.com/pyconindia/
   icon: assets/images/twitter-logo.svg
+
+- title: mailing-list
```

Comment:
Can you remove this until we have a logo ? It will show a broken image otherwise. I have asked to get the logos done by design team ?